pkg/mgrconfig/load.go

```diff
@@ -94,12 +94,49 @@ func defaultValues() *Config {
 		Procs:          6,
 		PreserveCorpus: true,
 		Experimental: Experimental{
-			RemoteCover: true,
-			CoverEdges:  true,
+			RemoteCover:      true,
+			CoverEdges:       true,
+			DescriptionsMode: DescriptionsMode(ManualDescriptions).String(),
```

Comment:
This looks like overly complex way to assign default value. String method is needed just here.

If  you create a const for "manual":
```
const manualDescriptions = "manual"
```
then you can use it both here and in strToEnum.

Reply:
Done!